docs/manual/introduction.tex

```diff
@@ -721,6 +721,10 @@
   JDK, but could also be useful when the warning/note is redundant for a
   user.
   See comments for \<-ApermitUnsupportedJdkVersion> option above.
+\item \<-AcheckRedundantAnnotations>
```

Comment:
Did we already brainstorm option names?
What do you think of `warnRedundantAnnotations` (similar to `-AwarnUnneededSuppressions`)?
The other `-AcheckXXX` options are more about additional checks... but then again, what is the difference between additional checks and additional warnings? So what sounds better?

Reply:
I think `warnRedundantAnnotations` is more straightforward and easier to understand. `checkXX` doesn't say what we check for.